addons/account/models/res_config_settings.py

```diff
@@ -184,6 +189,11 @@ def _inverse_account_default_credit_limit(self):
                 setting.account_default_credit_limit,
                 self.company_id.id
             )
+    @api.depends('company_id')
+    def _compute_terms_url(self):
+        for record in self:
+            if record.module_website:
```

Comment:
hi @moerradi ,
do we need to do add a record.terms_url = '' before the if or in the else to prevent " Compute method failed to assign ... " error.

thanks

Reply:
Ah yes, thanks for the insight !

Reply:
@niyasraphy @moerradi I learnt recently that's not needed for stored field.
https://github.com/odoo/odoo/pull/121386#discussion_r1193876179